firmware/src/main.cpp

```diff
@@ -222,4 +228,7 @@ void loop()
 
   // handlet the current rotary position
   rotary.handle();
+
+  // regulate case temperature
+  caseTempController.caseTempController();
```

Comment:
Should this function not be little bit more selfdescribing? like "readTemperatureAndAdjustFan" or something

Reply:
Can do